.github/workflows/submitty_ci.yml

```diff
@@ -231,32 +231,29 @@ jobs:
               sudo python3 -m pip install --upgrade pip setuptools wheel
               pip3 -V
               sudo pip3 -V
-              pip3 install cryptography
+              # Testing specific
+              pip3 install selenium
+              pip3 install psycopg2
+              # General dependency
               pip3 install python-pam
               pip3 install "sqlalchemy<1.4.0"
-              pip3 install selenium
               pip3 install tzlocal
               pip3 install jsonschema
               pip3 install jsonref
-              pip3 install websocket_client
```

Comment:
This dependency is necessary for

https://github.com/Submitty/Submitty/blob/656f588461e028f3f7e596d1f486cf68646d4cc6/tests/e2e/base_testcase.py#L12

I'm guessing the E2E still works as some other dependency requires it as a transitive dependency?

Reply:
Okay, I've missed it here. 
Can you check for me if the pyPdf is used?

Reply:
It's not. That would be a typo and can be safely removed, as we use its successor `pyPDF2`.

Reply:
okay, that's the only dependency that matters then, I will only include the removal of pyPDF2.